src/arch/ppc64/include/arch/io.h

```diff
@@ -5,31 +5,69 @@
 
 #include <stdint.h>
 
+#define MMIO_GROUP0_CHIP0_LPC_BASE_ADDR 0x0006030000000000
+#define LPC_ADDR_START 0xC0000000
+#define LPCHC_IO_SPACE 0xD0010000
+#define LPC_BASE_ADDR (MMIO_GROUP0_CHIP0_LPC_BASE_ADDR + LPCHC_IO_SPACE - LPC_ADDR_START)
+
+/* Enforce In-order Execution of I/O */
+static inline void eieio(void)
+{
+	asm volatile("eieio" ::: "memory");
+}
+
 static inline void outb(uint8_t value, uint16_t port)
 {
+	uint8_t *l_ptr = (uint8_t *)(LPC_BASE_ADDR + port);
+	const uint8_t *i_ptr = (const uint8_t *)&value;
```

Comment:
I'm not sure how this is supposed to work... Function arguments on PPC64 are passed in registers, why do we have to save it in memory, obtain its address and then do a memory-to-memory copy? I think `*l_ptr = value;` would be more than enough. The same goes for other out* functions